test-support/src/main/java/org/apache/brooklyn/test/LogWatcher.java

```diff
@@ -182,7 +182,7 @@ public void doEncode(ILoggingEvent event) throws IOException {
                 }
                 LOG.trace("level="+event.getLevel()+"; event="+event+"; msg="+event.getFormattedMessage());
```

Comment:
Any reason we are not using the recommended syntax like `LOG.trace("level= {} ; event={} ; msg={}", event.getLevel(), event,event.getFormattedMessage() );` ?